pkg/cloudprovider/aws/fleet/packing/instancetype.go

```diff
@@ -0,0 +1,131 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package packing
+
+import (
+	"errors"
+	"sort"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// TODO get this information from node-selector
+var (
+	instanceTypesAvailable = []*nodeCapacity{
```

Comment:
Could you rename this to `nodeCapacityPools`, or `nodeCapacities`, or `nodePools` (probably the last one?) - or something like that?

Also, could we divide by architecture type (if it's premature to decide that, then that's ok, can wait for a later PR)?